server/utils/fieldValidation.js

```diff
@@ -166,27 +159,26 @@ const fieldOptions = {
     })
   ),
 
-  requiredStaffNeedingMedicalAttention: () =>
```

Comment:
There are no changes in here apart from moving away from declaring these as functions where possible